mod/InGameTracker/TrackerDescriptions.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Archipelago.MultiClient.Net.Models;
+using System;
+using System.Collections.Generic;
+
+namespace ArchipelagoRandomizer.InGameTracker
+{
+    /// <summary>
+    /// Class containing descriptions for the various items, put on its own for clarity
+    /// </summary>
+    public class TrackerDescriptions
+    {
+        public static void DisplayItemText(string itemID, ItemListWrapper Wrapper)
+        {
+            var inventory = Randomizer.SaveData.itemsAcquired;
+            Dictionary<string, Tuple<string, string>> hints = Randomizer.Tracker.Hints;
+            Wrapper.DescriptionFieldClear();
+            List<string> infos = new();
+            bool discoveredItem = Enum.TryParse(itemID, out Item result);
+            // There may be a few dummy items that aren't actually tracked by the randomizer, like FrequencyOWV
+            // In the event of a dummy item being submitted, we want to assume it does exist
+            // If it doesn't it'll run into the default condition
+            // Can't use the enums here for some reason, so use strings as the names
+            if (!discoveredItem || inventory[result] > 0)
```

Comment:
I think some of the "item" variables are getting confusing and requiring lots of comments because they're mixing up multiple concepts of "item".

How do you feel about `itemID` in this function becoming `trackerEntryID`, and then `discoveredItem` being something like `isAnAPItem`? Then the comment could be more like "Because we want to group signals together into one entry per frequency, there isn't a 1-to-1 mapping between AP items and tracker inventory entries".

Reply:
I might even go and redo how I do the Tracker Item entries; currently it pulls data from a few different Dictionaries. Might be better to just combine all this data into one struct.